cmd/ausoceantv/main.go

```diff
@@ -79,11 +92,14 @@ func main() {
 	ctx := context.Background()
 	app.setup(ctx)
 
-	// Serve static files when running locally
+	// Serve static files when running locally.
 	http.Handle("/s/", http.StripPrefix("/s", http.FileServer(http.Dir("s"))))
```

Comment:
By serving the static files from the go program, we lose hot reloading which is very handy during development. Instead I suggest you serve your static files from vite, and use vite's proxy feature to proxy the go server, so that they can all be served from the same origin during development.

Reply:
I was only changing the comment here to put a fullstop, but I agree. I think that the static file serving was implemented by @scruzin before Trek implemented Vite on this service, I imagine that it is mostly a copy of our other services.

I have made an issue to address this in a separate PR: #352 

In terms of the Vite proxy, does this allow for calls to ```/stripe``` or ```/api``` to be redirected to golang without having issues with CORS?

Reply:
Yes, sorry I just glanced at the changes briefly, I think I thought it was the line below you added. By proxying `/stripe` and `/api`  they will all be served from the same origin so CORS doesn't need to be considered during development. During production they either need to all be served from the same origin or you need to use the CORS headers